lib/onc_certification_g10_test_kit/tasks/generate_matrix.rb

```diff
@@ -204,83 +205,75 @@ def applicable_options(runnable)
         runnable_and_parents.map(&:suite_option_requirements).compact.flatten
       end
 
-      def generate_inferno_test_worksheet # rubocop:disable Metrics/CyclomaticComplexity
-        workbook.add_worksheet('Inferno Tests')
-        inferno_worksheet = workbook.worksheets[2]
+      def inferno_worksheet
+        workbook.worksheets[2]
+      end
 
-        columns = [
+      def columns # rubocop:disable Metrics/CyclomaticComplexity
+        @columns ||= [
           ['', 3, ->(_test) { '' }],
           ['', 3, ->(_test) { '' }],
```

Comment:
```suggestion
          ['Inferno Group', 13, ->(_test) { '' }],
```

A minor thing because it seemed a tad strange to me to not have a column name in here. I'm not sure I love how we flatten all groups into this column, but i can't think of an obvious better solution at the moment and this certainly works.

<img width="339" alt="Screenshot 2024-08-06 at 4 10 51 PM" src="https://github.com/user-attachments/assets/31511e9c-9dc5-4ae0-af72-5ede5ab13fb6">